genealogy/geneweb/Portfile

```diff
@@ -1,29 +1,48 @@
 PortSystem 1.0
+PortGroup           github 1.0
 
 name			geneweb
-version			5.02
-revision		3
+version			7.1-beta
```

Comment:
Is it possible for us to wait for a release version rather than use beta?

Reply:
We could. I've been frustrated that the version we have just doesn't compile.

Reply:
there is probably a released version that the port can be updated to?

Reply:
It doesn't look so good.

https://github.com/geneweb/geneweb/tags

Reply:
Giving it more thoughts, and looking at how versions are declared, I would favor committing this version to the tree. The software didn't have any 7.0 release and previous releases were numbered by hash values. 7.1-beta looks like a milestone but I'm not sure how active they are. By including 7.1-beta to MacPorts, we may only help them and we will definitely help the few users who mattered enough to file the trac tickets mentioned above.

I'm also willing to take open ownership now that I reformatted the messy Portfile :)

See the roadmap that is called "Expected roadmap for Geneweb 7.00":
https://geneweb.tuxfamily.org/wiki/TODO